test/unit/test-adapter.js

```diff
@@ -2,91 +2,63 @@
 
 var http = require('http');
 var assert = require('chai').assert;
-var proxyquire = require('proxyquire');
 var sinon = require('sinon');
 var nop = require('nop');
+var crypto = require('crypto');
+var Readable = require('stream').Readable;
 var getRandomPort = require('get-random-port');
+var errorCodes = require('../../dist/http-handler').errorCodes;
 var systemUnderTest = require('../../dist/adapter');
 var SlackMessageAdapter = systemUnderTest.default;
 var delayed = require('../helpers').delayed;
 
 // fixtures
-var workingVerificationToken = 'VERIFICATION_TOKEN';
+var workingSigningSecret = 'SIGNING_SECRET';
+var requestSigningVersion = 'v0';
+var workingRawBody = 'payload=%7B%22type%22%3A%22interactive_message%22%7D';
+var contentType = 'application/x-www-form-urlencoded';
+
+function createRequestSignature(signingSecret, ts, rawBody) {
```

Comment:
maybe this should be in the `test/helpers.js` file? then the crypto import would move with it too.